error_reporting/google/cloud/error_reporting/__init__.py

```diff
@@ -14,5 +14,32 @@
 
 """Client library for Stackdriver Error Reporting"""
 
+from google.cloud.error_reporting.client import Client, HTTPContext
 
-from google.cloud.error_reporting.client import Client
+
+def build_flask_context(request):
+    """Builds an HTTP context object from a Flask (Werkzeug) request object.
+
+     This helper method extracts the relevant HTTP context from a Flask request
+     object into an object ready to be sent to Error Reporting.
+
+     Example::
```

Comment:
Please manually call out that it is a Python code-block (rather than using `::` syntax). This relates to #2738